pylint/checkers/deprecated.py

```diff
@@ -73,6 +78,27 @@ def visit_import(self, node):
                 mod_name, class_name = name.split(".", 1)
                 self.check_deprecated_class(node, mod_name, (class_name,))
 
+    def deprecated_decorators(self) -> Container[str]:
+        """Callback returning the deprecated decorators.
+
+        Returns:
+            collections.abc.Container of deprecated decorator names.
+        """
+        # pylint: disable=no-self-use
+        return ()
+
+    @utils.check_messages(
+        "deprecated-decorator",
```

Comment:
```suggestion
        "deprecated-decorator"
```
(black magic comma, force the line break)